noir-projects/aztec-nr/aztec/src/macros/aztec.nr

```diff
@@ -13,12 +13,14 @@ use crate::{
 };
 
 /// Marks a contract as an Aztec contract, generating the interfaces for its functions and notes, as well as injecting
-/// the `process_log` and `sync_notes` functions PXE requires in order to discover notes.
+/// the `sync_notes` function PXE requires in order to discover notes.
```

Comment:
These docs were stale and I forgot to update them when streamlining the log processing.